sql/2021/caching/service_worker_rank.sql

```diff
@@ -0,0 +1,18 @@
+SELECT
+  client,
+  rank_magnitude AS rank,
+  COUNT(DISTINCT IF(feature = 'ServiceWorkerControlledPage', url, NULL)) AS sw_pages,
+  COUNT(DISTINCT url) AS total,
+  COUNT(DISTINCT IF(feature = 'ServiceWorkerControlledPage', url, NULL)) / COUNT(DISTINCT url) AS pct
+FROM
+  `httparchive.blink_features.features`,
+  UNNEST([1e3, 1e4, 1e5, 1e6, 1e7]) AS rank_magnitude
```

Comment:
Really not sure this syntax is easy to grok to be honest, even if it does format better than lots of zeros.